cli/crates/common/src/environment.rs

```diff
@@ -60,7 +94,8 @@ impl Environment {
                 user_dot_grafbase_path,
                 warnings: Vec::new(),
             })
-            .expect("cannot set environment twice");
+            .map_err(|_| ())
+            .expect("cannot) set environment twice");
```

Comment:
small typo here ("cannot)")

Reply:
I swear I've fixed this three different times now, I'm not sure why it keeps showing up haha